ProcessLib/BoundaryCondition/BoundaryCondition.cpp

```diff
@@ -49,6 +51,15 @@ BoundaryConditionBuilder::createBoundaryCondition(
                     integration_order, shapefunction_order, parameters);
     }
 
+    //
+    // Special boundary conditions
+    //
+    if (type == "Pressure")
```

Comment:
Still, I think Pressure may not be a good BC type name. There might be fluid pressure on boundaries in  coupled THM processes, which is Dirichlet type. It could be MechanicalPressure.